StackExchange.Redis/StackExchange/Redis/ConfigurationOptions.cs

```diff
@@ -78,6 +90,10 @@ internal static void Unknown(string key)
                         ConfigChannel = "configChannel", AbortOnConnectFail = "abortConnect", ResolveDns = "resolveDns",
                         ChannelPrefix = "channelPrefix", Proxy = "proxy", ConnectRetry = "connectRetry",
                         ConfigCheckSeconds = "configCheckSeconds", ResponseTimeout = "responseTimeout", DefaultDatabase = "defaultDatabase";
+#if !CORE_CLR
+            internal const string SslProtocols = "sslProtocols";
+#endif

```

Comment:
Same as above - just didn't have a way to validate that I changed the core clr portions of the code correctly, so I left it out.  Shouldn't be too hard to remove this if someone has the right build pieces locally.